pkg/metrics/metrics.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package metrics is for metrics, used in CSI
+package metrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Statistic is a common interface for histogram metrics
+type Statistic interface {
+	Collect() prometheus.Collector
+	EvaluateDuration(method string, start time.Time)
+}
+
+// VolumeMetrics is a structure, which encapsulate prometheus histogram structure. It used for volume operation metrics
+type VolumeMetrics struct {
+	VolumeOperationsDuration *prometheus.HistogramVec
+}
+
+// NewVolumeMetrics initializes volume metrics
+func NewVolumeMetrics() *VolumeMetrics {
+	vm := &VolumeMetrics{}
+
+	vm.VolumeOperationsDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "volume_operations_duration",
+		Help:    "Volume operations methods duration",
+		Buckets: prometheus.ExponentialBuckets(0.01, 1.5, 25),
```

Comment:
Could you please clarify why you choose this buckets size and why it is different here and for Partitions metrics?

Reply:
I set custom bucket according to my observations, I tested it with different buckets and decided to use these buckets as they are more suitable.